lib/code-renderer.js

```diff
@@ -0,0 +1,20 @@
+var React = require('react');
+var $ = React.createElement;
+
+var highlight = require('./highlight');
+
+module.exports = function codeRenderer(node, props, children) {
+  var lang = (node.info || '').trim() || null;
+  var className = 'language-' + lang;
```

Comment:
if you include a unique string in this `className`, we can then scope all the hljs styles to only apply to the code blocks

`devcards-syntax-highlighted-block` or similar.